packages/core/core/src/public/Bundle.js

```diff
@@ -116,7 +116,7 @@ export class MutableBundle extends Bundle implements IMutableBundle {
     this.#bundle.stats = stats;
   }
 
-  removeAsset(asset: Asset): void {
+  removeAsset(asset: Asset): ?NodeId {
```

Comment:
so far we don't expose nodes or node ids through the public api. Mind if I change this to not return the value of the inner `removeAsset`?

Reply:
Sure go ahead. I just did it to fix types